src/dialog/overflow-menu/overflow-menu.stories.ts

```diff
@@ -1,7 +1,18 @@
 import { storiesOf, moduleMetadata } from "@storybook/angular";
+import { withKnobs, boolean, number } from "@storybook/addon-knobs/angular";
 
 import { DialogModule } from "../../";
 
+let options;
+
+const createOptions = (count: number): Array<string> => {
+	if (options && count === options.length) {
+		return options;
+	}
+	options = Array(count).fill(0).map((x, i) => "Option " + (i+1));
+	return options;
+};
```

Comment:
```suggestion
}
```
to go with the function change.